### ISSUE TITLE
City State "Diplomatic missions are temporary more effective"

### ISSUE THREAD

**Issue by Vordikk** [LoneGazebo/Community-Patch-DLL#10336]
_1. Mod Version_
4.1, 4.2, happens in all of them anyway.

_4. Describe the Issue_
O hope that text is enough, as problem not related to very specific case but generally happens
City State quest-like condition "Diplomatic missions are temporary more effective" was supposed to provide more influence for diplomatic missions in vanilla, which included gifting units and gold to them
In VP gifting gold to CS is not allowed, so "Diplomatic missions are temporary more effective" condition does literally nothing. It does not affect bonus influence for gifting units either, and there's no any other diplomatic missions.
So this is not balance problem but just looks like bug, where such condition was forgotten to be completely removed.

I'm speaking of this one, to be clean. Influence bonus with all other CS ranges between 8-12 as well.
![Civ 5 Tests](https://github.com/LoneGazebo/Community-Patch-DLL/assets/146865968/2117c62b-8099-4d9f-bad7-0091034ba6a6)

**Comment by myclan123** [comment]
You can conduct the Diplomatic mission with your Diplomacy Units, maybe not a bug I suppose

**Comment by Vordikk** [comment]
> You can conduct the Diplomatic mission with your Diplomacy Units, maybe not a bug I suppose
Yes, it's also called "Diplomatic Mission" for some reasons, and yet it has no difference either, i've checked just now. Provides same amount once again.

**Comment by Vordikk** [comment]
@RecursiveVision well, actual bonus is increased (144 instead of 120 like on screenshot) but tooltip is not modified. Actual amount is unclean, and it's sometimes it's important to know exact amount, as well as to know if "diplomatic missions" window affected anyhow, and if not, then it should be given other name for sure.
![Civ 5 Tests](https://github.com/LoneGazebo/Community-Patch-DLL/assets/146865968/9daa8d9f-0968-4815-9d10-8cf634ebacd9)
